cmd/helm/push.go

```diff
@@ -54,5 +54,9 @@ func newPushCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 		},
 	}
 
+	f := cmd.Flags()
+	f.BoolVar(&client.InsecureSkipTLSverify, "insecure-skip-tls-verify", false, "skip tls certificate checks for the chart upload")
```

Comment:
Should this just be `--insecure` ?

Reply:
I see the `helm pull` use `insecure-skip-tls-verify`, so i add this flag in `helm push`, should i change it?

Reply:
no, lets keep like this